packages/calling-stateful-client/src/CallClientState.ts

```diff
@@ -92,6 +98,42 @@ export interface CaptionsInfo {
   spokenText?: string;
 }
 
+/* @conditional-compile-remove(rtt) */
+/**
+ * @beta
+ */
+export interface RealTimeTextInfo {
+  /**
+   * The sequence id of the real time text.
+   */
+  id: number;
```

Comment:
Is there a reason we went with the nomenclature id and not sequenceId here?
id may be confused with css id.

Reply:
To match with the syntax used for captions and RTT in the component layer.